contributing/writing-tests.md

```diff
@@ -0,0 +1,200 @@
+# Writing Acceptance Tests
+
+Terraform includes an acceptance test harness that does most of the repetitive
+work involved in testing a resource. For additional information about testing
+Terraform Providers, see the [Extending Terraform documentation](https://www.terraform.io/docs/extend/testing/index.html).
+This guide is meant to augment that documentation with information specific to the HCP provider.
```

Comment:
I might word this a little more strongly: `We suggest reading this documentation first`. I originally read this doc assuming that it would tell me everything I'd need to know to start writing acceptance tests.

Reply:
Ah yea, that definitely makes sense, will update